server.js

```diff
@@ -47,49 +50,90 @@ const CONFIG = {
       sub3: `${SPECIAL_DELIM}header${SPECIAL_SEP}x-source${SPECIAL_DELIM}`,
       cu: `${SPECIAL_DELIM}header${SPECIAL_SEP}x-target-url${SPECIAL_DELIM}`
     }
-  }
+  },
+  weather_conditions: {
+    url: process.env["WEATHER_CONDITIONS_URL"],
+    query: {
+      apikey: process.env["WEATHER_KEY"],
```

Comment:
What's the process for getting these new environment variables added to the deployed server?